go.sum

```diff
@@ -1,4 +1,6 @@
+cloud.google.com/go v0.26.0 h1:e0WKqKTd5BnrG8aKH3J3h+QvEIQtSUcf2n5UZ5ZgLtQ=
```

Comment:
This seems unrelated given it is a different package. Is it needed? Can we revert changes to this file if they are not related?